src/main/webapp/css/main.css

```diff
@@ -120,56 +114,13 @@ button {
   overflow-y: scroll;
 }
 
-/* This is for the settings dropdown. */
-#dropdown-settings {
-  display: inline-block;
-  position: relative
+.headerContainer {
```

Comment:
Let's add a `margin-top` here, I noticed it's a little too close to the navbar.

Reply:
Updated the pictures. Let me know if this isn't what you had in mind.

Reply:
Yes, this looks better, thanks!